# linear and angular velocity gains, eq. 20, paper
lambda_v: [1.0, 1.0, 1.0]
lambda_w: [1.0, 1.0, 1.0]
